common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3580,6 +3580,23 @@
     ],
     "sqlState" : "42805"
   },
+  "PARQUET_CONVERSION_FAILURE" : {
+    "message" : [
+      "Unable to create a Parquet converter for the data type <dataType> whose Parquet type is <parquetType>."
+    ],
+    "subClass": {
+      "WITHOUT_DECIMAL_METADATA" : {
+        "message" : [
+          "Unable to create a Parquet converter for <typeName> whose Parquet type is <parquetType> without decimal metadata. Please read this column/field as Spark BINARY type."
```

Comment:
I don't need to repeat parent class error message otherwise users will see duplicates.